src/utils/middlewares/azure_api_auth.ts

```diff
@@ -269,7 +271,8 @@ type AzureAllowBodyPayloadMiddlewareErrorResponses =
  */
 export const AzureAllowBodyPayloadMiddleware = <S, A>(
   pattern: t.Type<A, S>,
-  allowedGroups: ReadonlySet<UserGroup>
+  allowedGroups: ReadonlySet<UserGroup>,
+  notAllowedMessage = "User has no valid scopes: You are not part of any valid scope, you should ask the administrator to give you the required permissions."
```

Comment:
Maybe it's useful to explain that the error is generated by the payload.
What about

`"No valid scopes: You are not allowed to send such payloads. Ask the administrator to give you the required permissions."`

What do you think @fabriziopapi  @AleDore ?

Reply:
Yess I agree